dags/taar_amodump.py

```diff
@@ -48,6 +48,21 @@
     dag=dag
 )
 
+editorial_whitelist = EMRSparkOperator(
+    task_id="taar_update_whitelist",
+    job_name="Generate a JSON blob from editorial reviewed addons for TAAR",
+    execution_timeout=timedelta(hours=1),
+    instance_count=1,
+    owner="vng@mozilla.com",
+    email=["mlopatka@mozilla.com", "vng@mozilla.com"],
+    env=mozetl_envvar("taar_update_whitelist",
+                      {},
+                      {'MOZETL_SUBMISSION_METHOD': 'spark'}),
```

Comment:
`MOZETL_SUBMISSION_METHOD` should be set to `spark` by default, and will be ignored by `MozDatabricksRunSubmit` if it's every moved over.